src/tiles/list/tile-list-compact.ts

```diff
@@ -1,22 +1,208 @@
-import { css, html, LitElement } from 'lit';
+import { css, html, LitElement, nothing } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
+import { SortParam } from '@internetarchive/search-service';
+import DOMPurify from 'dompurify';
 import { TileModel } from '../../models';
+import { formatCount, NumberFormat } from '../../utils/format-count';
+import { formatDate, DateFormat } from '../../utils/format-date';
+import '../../mediatype-icon';
 
 @customElement('tile-list-compact')
 export class TileListCompact extends LitElement {
   @property({ type: Object }) model?: TileModel;
 
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Number }) currentHeight?: number;
+
+  @property({ type: Object }) sortParam?: SortParam;
+
   render() {
     return html`
-      <h1>${this.model?.title}</h1>
-      <h2>${this.model?.datePublished?.toDateString()}</h2>
+      <div id="list-line" class="${this.classSize}">
+        <div id="thumb">${this.imageTemplate}</div>
+        <div id="title">${DOMPurify.sanitize(this.model?.title ?? '')}</div>
+        <div id="date">${formatDate(this.date, this.formatSize)}</div>
+        <div id="creator">${DOMPurify.sanitize(this.model?.creator ?? '')}</div>
+        <div id="views">
+          ${formatCount(this.model?.viewCount ?? 0, this.formatSize)}
+        </div>
+        <div id="icon">
+          <mediatype-icon
+            .mediatype=${this.model?.mediatype}
+            style="--iconHeight: 20px; --iconWidth: 20px;"
```

Comment:
Instead of inlining this, could you put this in the `static get styles()` section? ie:
```
mediatype-icon {
  --iconHeight: 20px; 
  --iconWidth: 20px;
}
```

Reply:
done